docs/content/developers/building-contributing.md

```diff
@@ -26,70 +26,86 @@ Each [PR build](https://github.com/ddev/ddev/actions/workflows/pr-build.yml) cre
 
 Download and unzip the appropriate binary and place it in your `$PATH`.
 
-## Rollback to a previous version
+!!!tip "You can also [downgrade to an older version of DDEV](../users/usage/faq.md#how-can-i-install-a-specific-version-of-ddev) (perform a rollback)."
 
-You can also [downgrade to an older version of DDEV](../users/usage/faq.md#how-can-i-install-a-specific-version-of-ddev).
+=== "Homebrew with macOS or Linux"
```

Comment:
At first i've thought this PR would have added the entire "homebrew with macos or linux" section but turns out after having the current and the new version side by side in preview that this section already existed. tbh  both sections ("homebrew  with macos or linux" and "installing a downloaded binary in the path" are sort of overlapping and doing basically the same. for me as a none technical user having more than one option where each is doing basically the same is rather stalling and freezing instead of providing clear and actionable instructions. i would rather suggest to drop the homebrew section entirely? plus the keyword homebrew is creating false expectations. my expectation based on the title would be the install of the PR binary would be based on homebrew but the sole reason why homebrew is named there is for unlinking ddev and linking it again after the testing is done.  plus by placing the binary in `/usr/local/bin/` sudo is needed. the other approach doesnt need it which would be preferable imho.

Reply:
> sort of overlapping and doing basically the same.

I agree.

> plus by placing the binary in `/usr/local/bin/` sudo is needed. the other approach doesnt need it which would be preferable imho.

I want to mention that if you are using PhpStorm, the binary is initially saved for each project.

So if you have the original install at `/usr/local/bin/ddev` (or have a symlink here), add override it with `~/bin/ddev`, and open the project with PhpStorm, it will still use `/usr/local/bin/ddev` for its tasks, which is frustrating (because it will re-download `~/.ddev/bin/docker-compose` all the time).

---

Ideally I prefer to replace the original DDEV binary, and make a backup of the old one:

```
# Make yourself the binary owner and do a backup
binary="$(readlink -f "$(which ddev)")"
sudo chown "${USER}:${USER}" "${binary}"
cp "${binary}" "${binary}.bak"

# Download ZIP file using wget, curl, dl, or using a browser and unzip it
unzip ddev.zip
chmod +x ddev && mv ddev "${binary}"

# Do the testing, and to go back, do the following:
binary="$(readlink -f "$(which ddev)")"
cp "${binary}.bak" "${binary}"
```

I think this will be better in terms of interaction with PhpStorm.

I am already using this technique myself, happy with it, and I can call the original DDEV with `ddev.bak` at the same time.

Let's replace the Homebrew section with this technique.